### ISSUE TITLE
Reuseable content nodes

### ISSUE THREAD

**Issue by z3r0** [symfony-cmf/symfony-cmf#18]
Content should be readable/writeable by different bundles. For example a news post can be rendered in a news ticker, in a blog, via an aggregated site wide RSS feed.

Furthermore content nodes can be combined. Aka a blog post can reference images, videos or even a standard content piece (like a disclaimer). This should be nestable although users should be guided to not create structures that are too deep. Bundles may define drop zones to allow constructing new content based on existing nodes.

In other words the content shouldn't be accessible or controlled from just one bundle. Bundle's should assume that other bundles can also access and modify the content.
